vulcan/lib/client/jsx/components/workflow/session/primary_inputs.tsx

```diff
@@ -29,7 +30,7 @@ export default function PrimaryInputs() {
               workflow.inputs[name].default
           ].find(a => a != null)
     }))
-  }, []);
+  }, [session]);
```

Comment:
Interesting find.  This memo should technically also include `workflow`.  It should read `[workflow, session]` since both of those values are used from the closure of the inner useMemo function.